integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -68,10 +68,17 @@ spec:
       {{- if .Values.tolerations }}
 {{ toYaml .Values.tolerations | trim | indent 8  }}
       {{- end }}
+    {{- if eq .Values.worker.fuseEnabled true }}
+      securityContext:
+        runAsUser: {{ .Values.fuse.user }}
+        runAsGroup: {{ .Values.fuse.group }}
+        fsGroup: {{ .Values.fuse.fsGroup }}
+    {{- else }}
       securityContext:
         runAsUser: {{ .Values.user }}
         runAsGroup: {{ .Values.group }}
         fsGroup: {{ .Values.fsGroup }}
+    {{- end }}
```

Comment:
+1 to @ZhuTopher , let's err out when a discrepancy is spotted, instead of letting the container run into permission issues at runtime.

Also don't forget to add to the comments, as previously mentioned :)